components/icon/Icon.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Author: bladimirardiles
+ * Component File Name: icon.js
+ * Component Name: icon
+ * Project: design-system-component-library-football-fanatics
+ * Date: Mon 16/10/2023 - 13:14
+ */
+
+import React, { useMemo } from 'react';
+import './icon.css';
+import { EIconColors, EIconSizes, EIConTypes } from './icon.enum';
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  ArrowDownIcon,
+  ArrowUpIcon,
+} from '@heroicons/react/24/solid';
```

Comment:
We created an issue to replace this import with the button from the figma design: https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/4